### ISSUE TITLE
Loss isn't convergent

### ISSUE THREAD

**Issue by blossomzx** [YipengHu/label-reg#7]
I use the example data to train this networks and find loss isn't convergent. Why is this happening?

**Comment by YipengHu** [comment]
You have to provide more details. Did you use default config file? Any
difference from the demo?

**Comment by zxblossom** [comment]
Yes,I did. I didn't change any content in that config file.

**Comment by YipengHu** [comment]
Thanks. What did you mean by "it didn't converge"? The loss didn't go down?

**Comment by blossomzx** [comment]
Converge means that the loss is always go down.But the loss in the example data sometimes would be bigger than the last time.

**Comment by blossomzx** [comment]
I'm so sorry that my expression is not very professional.

**Comment by blossomzx** [comment]
Thanks for your reply.

**Comment by YipengHu** [comment]
Sorry I'm still a bit confused. What does your loss look like at the
beginning of training? And what's it like after a few thousands iterations?
Remember this is based on stochastic gradient descent, the loss going down
is not guaranteed for each iteration.

**Comment by blossomzx** [comment]
Thanks, I understand it.
